telluric/georaster.py

```diff
@@ -734,6 +735,19 @@ def __contains__(self, geometry):
         """
         return self.footprint().contains(geometry)
 
+    def band(self, band_name):
```

Comment:
doesn it do the same as `_subimage`?

Reply:
if we do need both method, at least use subimage in this one so will not do the band index calculation in two places

Reply:
ok, I will rename `subimage ` which is to general to `bands_data`